### ISSUE TITLE
Sass vs. Compass

### ISSUE THREAD

**Issue by hectcastro** [azavea/docker-static-core#3]
Determine if there is consensus on the design team for Sass or Compass so we can package those dependencies into the container as well.

/cc @jfrankl @designmatty @ctaylo37

**Comment by jfrankl** [comment]
We all use Sass. My understanding is that Compass is additional frontend development tools that are built on top of Sass. I don't think any of us use Compass.

In terms of compiling Sass, we are very interested in the LibSass project—a C/C++ Sass compiler that is very fast. If you can make that work, I think it would be great for us to use it.

**Comment by ctaylo37** [comment]
+1

**Comment by designmatty** [comment]
+1

**Comment by rajadain** [comment]
AFAIK LibSass is used in Stormwater Interactive, not sure if MMW uses that or the regular Ruby compiler.

**Comment by caseypt** [comment]
Yeah, any project that uses [grunt-sass](https://github.com/sindresorhus/grunt-sass) is using libsass under the hood. The newer USACE projects use it as well, in case you wanted to check the performance out.

**Comment by jfrankl** [comment]
Any reason to not use grunt-sass/libsass on all projects?

**Comment by notthatbreezy** [comment]
No I don't think so, MUB generated a template using Yeoman that was using `grunt-compass`; I think I'd actually prefer not having ruby and compass installed so this is all good.

We're just getting started generating some base images with the intention that they be used across the company on different projects similar to how our ansible roles are used. The issue was opened to determine if there's a consensus, which seems like there is.

**Comment by hectcastro** [comment]
> Any reason to not use grunt-sass/libsass on all projects?

Early in the NYC project, some versions of NPM were having a hard time building the dependencies required for `grunt-sass` to do its thing. This led to build instability, and ultimately forced us toward the more stable Ruby version of Sass.

Crafting images that are known to work and reusing them can help mitigate these types of issues. The goal is to build all of those dependencies once and distribute vs. rebuilding dependencies inside of each person's VM (more opportunities for failure).
